scripts/npm-audit-comparison.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o xtrace
+
+# Run npm audit on current branch and compare it with the results of running npm audit on the base branch that is set in the package.json. If there are more
+# high or critical findings in the current branch, then the test fails. If the same number of findings are found, then check that the vulnerabilities are the same.
+# If they are are different, then the test fails.
+# Test is optional to merge. Use your and the reviewer's best judgement
```

Comment:
Specific comment feels weird to me since we decide this and set it in the GitHub UI anyway